### PR TITLE
Comment creation box profile pic shows/hides on focus/blur like facebook

### PR DESCRIPTION
**Pull request by ProCynic**
Not perfect yet, still need hide on comment submit.

### CODE REVIEW

File: conf/routes [hunk 1]
```diff
@@ -54,7 +58,6 @@ GET     /application.next.gif                   staticFile:/public/images/next.g
 GET     /application.calendar.jpg               staticFile:/public/images/calendar.gif
 
 # Photos
-GET     /users/{<[0-9]+>ownerId}/photos/?       Photos.photos
```
Comment (amshali) [inline conf/routes:57]: because we don't need to show photos from an arbitrary user?

File: conf/routes [hunk 2]
```diff
@@ -110,13 +112,11 @@ GET     /comments/{<[0-9]+>commentId}                   Comments.comment #Get a
 DELETE  /comments/{<[0-9]+>commentId}                   Comments.deleteComment #Delete a comment
 GET     /{Commentable}/{<[0-9]+>statusId}/comments/?           Comments.comments #Get all comments for a status
 POST    /{Commentable}/{<[0-9]+>statusId}/comments/?           Comments.addComment #Add a comment to a status
-GET     /users/{<[0-9]+>userId}/comments/?              Comments.userComments #get all comments by a user
 
 #Events
 GET     /events/?                                       Events.events #all events
 GET     /events/upcoming/?                              Events.upcoming #upcoming events
 GET     /events/past/?                                  Events.past #past events
-GET     /users/{<[0-9]+>userId}/events/?                Events.events #1 user's events
```
Comment (amshali) [inline conf/routes:119]: why removing this?

Comment (ProCynic) [inline conf/routes:119]: I moved it up to the users section

File: conf/routes (hunk 1 above)
Comment (ProCynic) [inline conf/routes:57]: line 27

File: conf/routes (hunk 2 above)
Comment (amshali) [inline conf/routes:119]: cool

**Comment by ProCynic** [comment]
Solved #167 for posts.  May be able to use `@NamedQuery` to generalize it to likeable and have every model implement its own findVisible method.  Only Post for now.

**Pull request merged**